bus/routes.go

```diff
@@ -2018,9 +2018,9 @@ func (b *Bus) webhookHandlerPost(jc jape.Context) {
 }
 
 func (b *Bus) metricsHandlerDELETE(jc jape.Context) {
-	metric := jc.PathParam("key")
-	if metric == "" {
-		jc.Error(errors.New("parameter 'metric' is required"), http.StatusBadRequest)
+	key := jc.PathParam("key")
```

Comment:
I wonder if this is even possible considering it's a path param. If `key` is `""`, then the path would be different and the router would probably return a 404 before even calling the handler.